articles/iot-operations/get-started/quickstart-upload-telemetry-to-cloud.md

```diff
@@ -86,6 +86,11 @@ Your codespace comes with the following sample connector configuration file, `/w
 
 1. Save the file.
 
+1. Navigate to the correct directory. Run the following command:
```

Comment:
I don't think this is necessary - the next command explicitly includes the path to the file:

` kubectl apply -f samples/quickstarts/datalake-connector.yaml`